protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1283,16 +1271,18 @@ message ObserveCandleDataResponse {
 // Request that is used when listing candles for a market at an interval
 message ListCandleDataRequest {
     // Candle ID, required field.
-    string candle_id = 1;
+    string candle_id = 1 [(google.api.field_behavior) = REQUIRED];
 
     // Timestamp to retrieve candles since, in nanoseconds since the epoch, required field
-    int64 from_timestamp = 2;
+    int64 from_timestamp = 2 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Even though in the doc-string it says from/to timestamps are required the truth is they are not:
```
https://api.n06.testnet.vega.xyz/api/v2/candle?candleId=trades_candle_5_minutes_10c7d40afd910eeac0c2cad186d79cb194090d5d5f13bd31e14c49fd1bded7e2
```
maybe they should be required true? I don't know.

Reply:
I don't think they should be required, it looks like the implementation allows the user to have an open ended result set, e.g. candlesticks since T, or candlesticks until T, as well as candlesticks between T and T' and the doc-string is incorrect. I'll remove the required status and update the doc string.

Reply:
Addressed https://github.com/vegaprotocol/vega/pull/8070/commits/16a86a18b2d74fe341c6e3d4e19eb7563618cc95